packages/base/source/server/errors.js

```diff
@@ -12,4 +11,4 @@ Space.Error.extend(Todos, 'TodoNotFoundError', {
     Space.Error.constructor.call(this);
     this.message = `No todo with id ${todoId} found.`;
```

Comment:
Did you try this as Dominik suggested, by supplying the message as the second param to the Space.Error.constructor call?